src/map/script.c

```diff
@@ -11591,14 +11591,16 @@ BUILDIN(getstatus)
 		case 3: script_pushint(st, sd->sc.data[id]->val3); break;
 		case 4: script_pushint(st, sd->sc.data[id]->val4); break;
 		case 5:
-		{
-			const struct TimerData *td = timer->get(sd->sc.data[id]->timer);
+			if (sd->sc.data[id].infinite_duration) {
```

Comment:
Ah, I didn't noticed, was in hurry and with no build environment :x
Yup, I know error was due to no return value, but infinite timers would trigger that ShowError part. And it was sensible to return -1 instead